vertical-pod-autoscaler/recommender/base/histogram.go

```diff
@@ -0,0 +1,82 @@
+package base
+
+// Histogram represents an approximate distribution of some variable.
+type Histogram interface {
```

Comment:
You mean the total weight of all samples added to the histogram?
Shall we replace Empty() ? (it will become redundant).